INSTALL

```diff
@@ -56,6 +68,10 @@ How to install PBS Pro using the configure script.
   following command:
 
     ./configure --prefix=/opt/pbs --libexecdir=/opt/pbs/libexec
+    
+  For Debian systems you should run the following command:
+
+    ./configure
```

Comment:
This should be the same as for CentOS. Please consolidate Debian and CentOS instructions for calling configure. For both, we want "./configure --prefix=/opt/pbs". Thanks for your patience, that's the last issue I have.